glastopf/modules/handlers/emulators/surface/create_surface.py

```diff
@@ -23,11 +23,26 @@
 
 
 class SurfaceCreator(base_emulator.BaseEmulator):
-    def __init__(self, data_dir):
+    def __init__(self, data_dir, conf_parser=None):
         super(SurfaceCreator, self).__init__(data_dir)
+        self.conf_parser = conf_parser
         self.template_env = Environment(loader=FileSystemLoader(os.path.join(self.data_dir, "templates")))
 
     def get_index(self, title="Title Title", target="/index", body="Some Body", footer="Footer Text"):
         template = self.template_env.get_template('index.html')
-        surface_page = template.render(title=title, target=target, body=body, footer=footer)
+        google_meta = ""
+        bing_meta = ""
+        try:
+            meta_g = self.conf_parser.get('surface', 'google_meta')
+            meta_b = self.conf_parser.get('surface', 'bing_meta')
+            if meta_g:
+                google_meta = "<meta name=\"google-site-verification\" content=\"%s\" />" % meta_g
```

Comment:
As far as I can see, we're only substituting the string here, so it makes sense to do that inside the template. After all, that's what the templates are for!